cmd/server.go

```diff
@@ -91,7 +91,7 @@ func RunServer(cmd *cobra.Command, args []string) { // coverage-ignore
 	shutdownCtx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
 	defer cancel()
 
-	err = server.Shutdown(shutdownCtx)
+	err = srv.Shutdown(shutdownCtx)
 	if err != nil {
 		log.Panic().Err(err).Msg("Graceful shutdown failed")
```

Comment:
Not part of the PR but I don't think we should log a panic in case of a failed shutdown (normal error would do)